tests/main_test.ts

```diff
@@ -44,6 +45,14 @@ Deno.test("/ page prerender", async () => {
     body,
     '<meta name="description" content="Hello world!"/>',
   );
+  assertStringIncludes(
+    body,
+    '<meta name="generator" content="The freshest framework!"/>',
+  );
+  if (body.includes("specialTag")) {
```

Comment:
I didn't see a negation version of `assertStringIncludes` (something like `assertStringDoesntInclude`), so I did it like this. Suggestions?

Reply:
You can use `assert()` for plain boolean checks. Example:

```js
assert(foo === bar, "My error message when the check fails")

// your example
assert(!body.includes("specialTag"), `Expected actual: "${body}" to not contain: "specialTag"`);
```